lib/css/components/links.less

```diff
@@ -1,214 +1,102 @@
 a,
 .s-link {
-    text-decoration: none;
-    color: var(--theme-link-color);
-    cursor: pointer;
-    user-select: auto;
+    --_li-fc: var(--theme-link-color);
+    --_li-fc-hover: var(--theme-link-color-hover);
+    --_li-fc-visited: var(--theme-link-color-visited);
 
+    // CONTEXTUAL STYLES
     .highcontrast-mode({
         text-decoration: underline;
     });
 
-    &.s-link__visited:visited {
-        color: var(--theme-link-color-visited);
-    }
-
-    &:hover,
-    &.s-link__visited:hover,
-    &:active,
-    &.s-link__visited:active {
-        color: var(--theme-link-color-hover);
-    }
-
-    &.s-link__grayscale {
-        color: var(--black-800);
+    // STATES
+    &.s-link {
+        // MODIFIERS
+        &__dropdown {
+            &:after {
+                border-width: var(--su-static4); // this needs to come before any below border modifications
+                border-bottom-width: 0;
+                border-color: currentColor transparent;
+                border-style: solid;
+                border-top-width: var(--su-static4);
+                content: "";
+                pointer-events: none;
+                position: absolute;
+                right: 0;
+                top: calc(50% - var(--su-static2));
+                z-index: var(--zi-active);
+            }
 
-        &.s-link__visited:visited {
-            color: var(--black-700);
+            padding-right: 0.9em;
+            position: relative;
         }
 
-        &:hover,
-        &.s-link__visited:hover,
-        &:active,
-        &.s-link__visited:active {
-            color: var(--black-900);
+        &__underlined {
+            text-decoration: underline !important;
         }
-    }
 
-    &.s-link__muted {
-        color: var(--black-500);
-
-        &.s-link__visited:visited {
-            color: var(--black-700);
+        &__visited { // TODO can we get rid of this? It's only in 3 places in Core and seems to me it doesn't even serve a purpose.
+            --_li-fc-hover: var(--theme-link-color-hover);
+            --_li-fc-visited: var(--theme-link-color-visited);
```

Comment:
I _think_ we could get rid of this. Keeping it around for now just for parity with the old CSS.